kerykeion/charts/draw_planets.py

```
@@ -832,7 +827,7 @@ def _draw_external_natal_lines(
         + f'style="stroke-width:1px;stroke:{color};stroke-opacity:.3;"/>\n'
         + f'<line x1="{x2}" y1="{y2}" x2="{x3}" y2="{y3}" '
         + f'style="stroke-width:1px;stroke:{color};stroke-opacity:.5;"/>\n'
-        + f'</g>'
+        + f"</g>"
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)